src/vellum/workflows/nodes/displayable/subworkflow_deployment_node/node.py

```diff
@@ -70,8 +70,10 @@ def _compile_subworkflow_inputs(self) -> List[WorkflowRequestInputRequest]:
                         value=input_value,
                     )
                 )
-            elif isinstance(input_value, list) and all(
-                isinstance(message, (ChatMessage, ChatMessageRequest)) for message in input_value
+            elif (
+                isinstance(input_value, list)
+                and len(input_value) > 0
+                and all(isinstance(message, (ChatMessage, ChatMessageRequest)) for message in input_value)
```

Comment:
[oos] We'll need a vellum-side update that allows CHAT_HISTORY input variables to accept empty JSON `[]`s